map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -65,15 +68,21 @@ Lanelet2MapLoaderNode::Lanelet2MapLoaderNode(const rclcpp::NodeOptions & options
   // create map bin msg
   const auto map_bin_msg = create_map_bin_msg(map, lanelet2_filename, now());
 
+  const auto map_projector_type_msg = get_map_projector_type(
+    lanelet2_filename, lanelet2_map_projector_type, map_origin_lat, map_origin_lon);
   // create publisher and publish
   pub_map_bin_ =
     create_publisher<HADMapBin>("output/lanelet2_map", rclcpp::QoS{1}.transient_local());
   pub_map_bin_->publish(map_bin_msg);
+  // create publisher and publish
+  pub_map_projector_type_ =
+    create_publisher<MapProjectorInfo>("map_projector_type", rclcpp::QoS{1}.transient_local());
+  pub_map_projector_type_->publish(map_projector_type_msg);
 }
 
 lanelet::LaneletMapPtr Lanelet2MapLoaderNode::load_map(
```

Comment:
(correct me if I am wrong)
Since this function does not use member variables, how about making it independent function from `Lanelet2MapLoaderNode` class?